res/controllers/Hercules_DJControl_Inpulse_200.midi.xml

```diff
@@ -0,0 +1,1648 @@
+                <key>DJCi200.wheelTouch</key>
+                <description>Jog Wheel Touch Deck B</description>
+                <status>0x92</status>
+                <midino>0x08</midino>
+                <options>
+                    <script-binding />
+                </options>
+            </control>
+            <!--Loop In/Out-->
+            <control>
+                <group>[Channel2]</group>
+                <key>beatloop_4_activate</key>
+                <description>Loop In button</description>
+                <status>0x92</status>
+                <midino>0x09</midino>
+                <options>
+                    <normal />
+                </options>
+            </control>
+            <control>
+                <group>[Channel2]</group>
+                <key>reloop_toggle</key>
+                <description>Loop Out button</description>
+                <status>0x92</status>
+                <midino>0x0A</midino>
+                <options>
+                    <normal />
+                </options>
+            </control>
+            <!-- NN's MIDI Channel 4 (0x93 - SHIFT MODE)-->
+            <!--Browser section (Encoder button)-->
+            <control>
+                <group>[Master]</group>
+                <key>maximize_library</key>
+                <description>Browser button - Maximize Library view</description>
+                <status>0x93</status>
+                <midino>0x00</midino>
+                <options>
+                    <normal />
+                </options>
+            </control>
+            <!-- NN's MIDI Channel 5 (0x94 Deck A - SHIFT MODE)-->
+            <!--Jog Touch A-->
+            <control>
+                <group>[Channel1]</group>
+                <key>DJCi200.wheelTouchShift</key>
+                <description>Jog Wheel Shift Touch Deck A</description>
+                <status>0x94</status>
+                <midino>0x08</midino>
+                <options>
+                    <script-binding />
+                </options>
+            </control>
+            <!--Play-->
+            <control>
+                <group>[Channel1]</group>
+                <key>play_stutter</key>
+                <description>SHIFT + Play: Play Stutter</description>
+                <status>0x94</status>
+                <midino>0x07</midino>
+                <options>
+                    <normal />
+                </options>
+            </control>
+            <!--CUE-->
+            <control>
+                <group>[Channel1]</group>
+                <key>start_play</key>
+                <description>SHIFT + Cue: REWIND to beginning</description>
+                <status>0x94</status>
+                <midino>0x06</midino>
+                <options>
+                    <normal />
+                </options>
+            </control>
+            <!--Sync-->
+            <control>
+                <group>[Channel1]</group>
+                <key>sync_master</key>
```

Comment:
@DJPhatso Are you actually using this? If yes, can you explain what you use it for? We're reworking master sync behavior in PR #2376 and think about dropping support for setting `sync_master` manually. (ping @ywwg)

Reply:
You mean personally ? No I don't, but I had seen it mapped on other controllers, so I figured it might be useful to someone. But given it is a "secondary" SHIFT function, you have to know that it is there.

Reply:
Yeah, maybe it makes sense to map it to toggle quantize or trigger beatsync instead then.

Reply:
Possible. Or remove it entirely. Then it would need to be changed in #2465 as well?

Reply:
Yes, but we don't have a definitive decision regarding #2376 yet.

Reply:
Any news on that ?

Reply:
Sorry, it's still being discussed here: https://mixxx.zulipchat.com/#narrow/stream/109171-development/topic/2.2E3.20planning.3A.20https.3A.2F.2Fgithub.2Ecom.2Fmixxxdj.2Fmixxx.2Fprojects.2F2/near/192204135

Reply:
Looks like we keep the ability to set sync master explicitly, so it's up to you if you want to keep this to use the button for something else (like toggling quantize) .

Reply:
Noted. I'll leave them as is now. 

Meanwhile, I made some changes to the the .xml to correct an issue I found and addressed an upcoming update to the device that will change how the pads behave in normal mode. This way, the mapping will work whether or not the users update their devices. I will add the relevant information in the description page once the update is released.